src/selectors/flavors.js

```diff
@@ -0,0 +1,8 @@
+import { createSelector } from 'reselect';
+
+export const getFlavors = state => state.flavors;
+
+export const getAllFlavors = createSelector(
+  getFlavors,
+  flavors => flavors.flavors
```

Comment:
Usually when I see a word duplicated like this in one of my selectors I try to refactor the redux state so that it reads cleanly, like `flavors.list` - not saying you should change it to that or that we should use `.list` as a pattern, just wanted to mention it.

Reply:
I refactored this to flavors.collection